tests/e2e_test/test_e2e.py

```diff
@@ -22,33 +22,107 @@
+            f"--disable-fixed-node-init-instance "
+            f"--initial-instances {instances_num} "
+            f"--enable-migration "
+            f"--model {model} "
+            f"--engine-use-ray "
+            f"--worker-use-ray "
+            f"--max-model-len {max_model_len} "
+            f"--dispatch-policy {dispatch_policy} "
+            f"--trust-remote-code "
+            f"--request-migration-policy LCFS "
+            f"--migration-backend {migration_backend} "
+            f"--migration-cache-blocks 32 "
+            f"--tensor-parallel-size 1 "
+            f"--request-output-queue-port {1234+port} "
+            f"{'--launch-ray-cluster ' if launch_ray_cluster else ''}"
+            f"{'> instance_'+result_filename if len(result_filename)> 0 else ''} 2>&1 &"
+        )
+    elif launch_mode == 'dief':
+        command = (
+            f"RAY_DEDUP_LOGS=0 HEAD_NODE_IP={HEAD_NODE_IP} HEAD_NODE=1 "
+            f"nohup python -m llumnix.entrypoints.vllm.api_server "
+            f"--host {ip} "
+            f"--port {port} "
+            f"--disable-init-instance-by-manager "
+            f"--initial-instances {instances_num} "
+            f"--enable-migration "
+            f"--model {model} "
+            f"--engine-use-ray "
+            f"--worker-use-ray "
+            f"--max-model-len {max_model_len} "
+            f"--dispatch-policy {dispatch_policy} "
+            f"--trust-remote-code "
+            f"--request-migration-policy LCFS "
+            f"--migration-backend {migration_backend} "
+            f"--migration-cache-blocks 32 "
+            f"--tensor-parallel-size 1 "
+            f"--request-output-queue-port {1234+port} "
+            f"{'--launch-ray-cluster ' if launch_ray_cluster else ''}"
+            f"{'> instance_'+result_filename if len(result_filename)> 0 else ''} 2>&1 &"
+        )
+    else: # launch_mode == 'didf':
+        command = (
+            f"RAY_DEDUP_LOGS=0 HEAD_NODE_IP={HEAD_NODE_IP} HEAD_NODE=1 "
+            f"nohup python -m llumnix.entrypoints.vllm.api_server "
+            f"--host {ip} "
+            f"--port {port} "
+            f"--disable-init-instance-by-manager "
+            f"--disable-fixed-node-init-instance "
+            f"--initial-instances {instances_num} "
+            f"--enable-migration "
+            f"--model {model} "
+            f"--engine-use-ray "
+            f"--worker-use-ray "
+            f"--max-model-len {max_model_len} "
+            f"--dispatch-policy {dispatch_policy} "
+            f"--trust-remote-code "
+            f"--request-migration-policy LCFS "
+            f"--migration-backend {migration_backend} "
+            f"--migration-cache-blocks 32 "
+            f"--tensor-parallel-size 1 "
+            f"--request-output-queue-port {1234+port} "
+            f"{'--launch-ray-cluster ' if launch_ray_cluster else ''}"
+            f"{'> instance_'+result_filename if len(result_filename)> 0 else ''} 2>&1 &"
+        )
```

Comment:
heavy code duplicate

Reply:
There are commands with different cli arguments.

Reply:
why not define the common part first then inject different arguments in different if-else branches?

Reply:
I think ues yaml to define to general config is better

Reply:
Get

Reply:
> I think using yaml to define to general config is better

I think yaml config should only be used by user, putting yaml config for e2e test is not a good choice.